compiler/rustc_hir_typeck/src/pat.rs

```diff
@@ -2065,11 +2069,28 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
         err.span_suggestion(
             sp,
             format!(
-                "if you don't care about {these} missing field{s}, you can explicitly ignore {them}",
-                these = pluralize!("this", len),
+                "if the value{s} {are} not relevant, discard {them} explicitly",
```

Comment:
I think the message did not need to be changed. Tweaking the suggestion was sufficient.

Reply:
Ok, reverted the message. https://github.com/rust-lang/rust/blob/10b60eba9bc55449721db912fe75bab11de43c4e/compiler/rustc_hir_typeck/src/pat.rs#L2068